tests/queries/0_stateless/02961_storage_config_volume_priority.reference

```diff
@@ -0,0 +1,4 @@
+vol2_02961	1
+vol1_02961	2
+vol_untagged1_02961	3
+vol_untagged2_02961	4
```

Comment:
I wonder why you expect vol_untagged1_02961 before vol_untagged2_02961?

Reply:
I do not. It is a file that was not commited